entries/n/netcorecloud.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Netcore Cloud": {
```

Comment:
```suggestion
  "Netcore Cloud Email API": {
```
2FA only applies to this specific product.

Reply:
why "Netcore Cloud Email API": when everywhere is "Netcore Cloud"?

Reply:
The doc page provided is for their email API:
![image](https://user-images.githubusercontent.com/20560218/124360066-9d888980-dc1f-11eb-8635-f666cf2047eb.png)

whereas other netcore cloud products (see https://netcorecloud.com/login) don't appear to support 2FA.